appinfo/Migrations/Version20170801085340.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace OCA\notifications\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use OC\DB\MDB2SchemaReader;
+use OCP\Migration\ISchemaMigration;
+
+class Version20170801085340 implements ISchemaMigration {
+
+	public function changeSchema(Schema $schema, array $options) {
+		$prefix = $options['tablePrefix'];
+		if ($schema->hasTable("{$prefix}notifications")) {
+			return;
+		}
+
+		// not that valid ....
+		$schemaReader = new MDB2SchemaReader(\OC::$server->getConfig(), \OC::$server->getDatabaseConnection()->getDatabasePlatform());
```

Comment:
why is this needed just for the icon ? separate PR for proper migration and kill XML file ?